### PR TITLE
upgrade go to 1.19

### PR DESCRIPTION
**Pull request by mangalaman93**

### CODE REVIEW

**Review by anurags92 — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -1,27 +1,37 @@
 module github.com/dgraph-io/badger/v3
 
-go 1.12
-
-// replace github.com/dgraph-io/ristretto => /home/amanbansal/go/src/github.com/dgraph-io/ristretto
```
Comment (nswamy) [inline go.mod:5]: Let's remove the specific person's and keep the comment

Comment (mangalaman93) [inline go.mod:5]: keep some sort of a pointer here about this

**Review by nswamy — APPROVED** [review]
please retain the comment otherwise LGTM

**Review by joshua-goldstein — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (mangalaman93) [reply go.mod:5]: Will add a reference in another PR. Thanks for taking a look.

**Pull request merged**